assets/CSS/style.css

```diff
@@ -0,0 +1,208 @@
+* {
+  box-sizing: border-box;
+  -webkit-box-sizing:border-box;
+  -moz-box-sizing:border-box;
+  font-family:Arial, Lucida, Lucida, Helvetica, Arial, Verdana, sans-serif;
+}
+.main-container {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  height:538px;
+  padding-top: 18px;
+  background-image: url("../images/your_verse_hero.jpg");
+  background-repeat: no-repeat;
+  background-size: 100% 1315px;
+  background-position: bottom;
+}
+.Global-Header {
+  max-width:960px;
+  display:flex;
+  height: 36px;
+  margin: 0px auto;
+  box-shadow: rgba(0,0,0,0.3) 0 1px 2px;
+  border-radius: 10px;
+  border: 1px solid #808080;
+}
+.Flex-Items {
+  flex:1;
+  text-align: center;
+  border-right: 1px solid #616161;
+  padding: 10px 0px;
+  color: white;
+  font-family: "Lucida", "Helvetica", " Verdana", sans-serif, "Arial";
+  font-size: 13px;
+  background: linear-gradient(#9a9a9a, #676767 50%, #808080);
+}
+.Flex-Items:hover {
+  background: #575757;
+}
+
+.flex-first{
+  border-top-left-radius: 10px;
+  border-bottom-left-radius: 10px;
+  padding: 0;
+}
+.flex-first img {
+  display: block;
+  margin: 5px auto;
+}
+.flex-last {
+  padding:0;
+  flex:1.3;
+  border: none;
+  border-top-right-radius: 10px;
+  border-bottom-right-radius: 10px;
+}
+.flex-last input {
+  background-image: url("../images/search.png");
+  background-repeat: no-repeat;
+  width:109px;
+  height: 19px;
+  background-position: 5px 1px;
+  background-size: 15px 14px;
+  background-color: transparent;
+  margin: 8px auto;
+  border-radius: 12px;
+  border-color: #adadad;
+  padding: 5px;
+  text-align: right;
+  color: white;
+}
+.Mid-Img-Container {
+  width: 980px;
+  height: 220.45px;
+  margin: 115px auto;
+}
+.mid-imgs {
+  margin-bottom: 16px;
+  width:980px;
+  height:58.67px;
+}
+.mid-imgs img{
+  display: block;
+  margin:0px auto;
+}
+
+.mid-first img {
+  width:680px;
+  height:54.67px;
+}
+
+.mid-second img {
+  width:980px;
+  height:82.14px;
+}
+.mid-third {
+  padding-top: 35px;
+}
+.mid-third img {
+  width:550px;
+  height:31.64px;
+}
+.bottom-img-holder {
+  /*positioning*/
+  position: absolute;
+  top:540px;
+  right: 0;
+  left: 0;
+  /*box-model*/
+  display:flex;
+  height:200px;
+  /*typography*/
```

Comment:
Always leave your CSS file free of unnecessary comments. If you don't need a style, rather than commenting it out, just remove it.